zebrad/src/application.rs

```diff
@@ -92,6 +92,7 @@ impl Application for ZebradApp {
         color_eyre::config::HookBuilder::default()
             .issue_url(concat!(env!("CARGO_PKG_REPOSITORY"), "/issues/new"))
             .add_issue_metadata("version", env!("CARGO_PKG_VERSION"))
+            .add_issue_metadata("git sha", ZebradCmd::SHA)
```

Comment:
Git uses `SHA1`s for a bunch of different objects. This is the `SHA1` of the git `HEAD` commit:
```suggestion
            .add_issue_metadata("git commit", ZebradCmd::SHA)
```